nginx/nginx.conf

```diff
@@ -0,0 +1,51 @@
+# You can adjust the configuration to add additional TLS or HTTP features.
```

Comment:
If you copied these nginx configs from server, please state so in a comment and indicate they need to be kept in sync with the upstream location. Otherwise, if they differ, that is fine as-is.